doc/man3/EVP_MAC.pod

```diff
@@ -60,25 +69,30 @@ B<EVP_MAC_CTX> is a context type that holds internal MAC information
 as well as a reference to a computation context, for those MACs that
 rely on an underlying computation algorithm.
 
+=head2 Algorithm implementation fetching
+
+EVP_MAC_fetch() fetches an implementation of a MAC I<algorithm>, given
+a library context I<libctx> and a set of I<properties>.
```

Comment:
Yeah, B is our own invention. I've had a look at the standards for writing man-pages, and they go for I. 

See http://man7.org/linux/man-pages/man7/man-pages.7.html

Reply:
Which suggests we need a PR to change all our existing man pages! Ouch.

Reply:
Yes.  Someone did submit such a PR a long while ago, I'm afraid it got lost in all the noise, and so much has changed since that it really needs to be redone.  Most of it can be automated, and I'm willing to do the work.

Reply:
What about a PR to doc-nits and fixes that catch their list of "do not use" phrases?  I can do that if desired.

Reply:
"do not use" phrases?  Not sure what you're after, but it's still allowed to use Both `B<` and `I<` for emphasis, so it's a bit difficult to know the author's intent.
(emphasis should probably not be needed in manuals, though)